s2clientprotocol/raw.proto

```diff
@@ -116,14 +120,14 @@ message Unit {
   optional int32 vespene_contents = 19;
   optional bool is_flying = 20;
   optional bool is_burrowed = 21;
+  optional bool is_hallucination = 38;
```

Comment:
Does that also reveal enemy hallucinations that are not in range of a detector? Because that is not what it should be in my opinion.

Reply:
Nope, it is true for your own or detected.